air/src/execution_step/value_types/canon_stream.rs

```diff
@@ -77,6 +77,14 @@ impl CanonStream {
     pub(crate) fn tetraplet(&self) -> &Rc<SecurityTetraplet> {
         &self.tetraplet
     }
+
+    pub(crate) fn push(&mut self, value_aggregate: &ValueAggregate) {
```

Comment:
it should be `ValueAggregate` without a reference because this functions requires it by a value.